streamingpro-mlsql/src/main/java/streaming/rest/RestController.scala

```diff
@@ -132,10 +134,24 @@ class RestController extends ApplicationController with WowLog {
                 skipGrammarValidate = paramAsBoolean("skipGrammarValidate", true))
 
               outputResult = getScriptResult(context, sparkSession)
-              htp.post(new Url(param("callback")),
-                Map("stat" -> s"""succeeded""",
-                  "res" -> outputResult,
-                  "jobInfo" -> JSONTool.toJsonStr(jobInfo)))
+
+              val maxTries = Math.max(0, paramAsInt("maxRetries", -1)) + 1
```

Comment:
Can we move  the retry logical out  of this controller? Also, we can use fluent api to replace the existing implementation.

Reply:
The logic has been moved out, also the API has been changed to HttpClient fluent.